contracts/libs/RollupUtils.sol

```diff
@@ -578,6 +613,24 @@ library RollupUtils {
         return Tx.serialize(_txs);
     }
 
+    function CompressManyTransferFromEncoded(bytes[] memory txBytes)
+        public
+        pure
+        returns (bytes memory)
+    {
+        Tx.Transfer[] memory _txs = new Tx.Transfer[](txBytes.length);
+        for (uint256 i = 0; i < txBytes.length; i++) {
+            Types.Transaction memory _tx = TxFromBytes(txBytes[i]);
+            _txs[i] = Tx.Transfer(
+                _tx.fromIndex,
+                _tx.toIndex,
+                _tx.amount,
+                _tx.nonce
+            );
+        }
+        return Tx.serialize(_txs);
```

Comment:
To compress many Transfers, we call this function

Reply:
So this is for N txs, right?

Reply:
Yes, the output is N transactions compressed into a bytes string.